components/dayspan-custom-calendar.vue

```diff
@@ -1,65 +1,79 @@
 <template>
   <v-layout column>
-    <v-flex pb-3>
-      <v-tooltip
-        v-bind="{setToday, todayDate, calendar}" 
-        name="today" 
-        bottom>
-        <v-btn 
-          slot="activator"
-          :icon="$vuetify.breakpoint.xs"
-          depressed
-          outline
-          @click="setToday">
-          <span v-show="$vuetify.breakpoint.smAndUp">{{ labels.today }}</span>
-          <v-icon v-show="!$vuetify.breakpoint.smAndUp">{{ labels.todayIcon }}</v-icon>
-        </v-btn>
-        <span>{{ todayDate }}</span>
-      </v-tooltip>
-
-      <v-tooltip 
-        v-bind="{setToday, todayDate, calendar}" 
-        name="today"
-        bottom>
-        <v-btn 
-          slot="activator"
-          icon 
-          depressed
-          @click="prev" >
-          <v-icon>keyboard_arrow_left</v-icon>
-        </v-btn>
-        <span>{{ prevLabel }}</span>
-      </v-tooltip>
-
-      <v-tooltip  
-        v-bind="{next, nextLabel, calendar}"
-        name="next" 
-        bottom>
-        <v-btn 
-          slot="activator"
-          icon 
-          depressed
-          @click="next">
-          <v-icon>keyboard_arrow_right</v-icon>
-        </v-btn>
-        <span>{{ nextLabel }}</span>
-      </v-tooltip>
-
-      <span
-        v-show="$vuetify.breakpoint.smAndUp"
-        v-bind="{summary, calendar}"
-        name="extendedDateSummary"
-        class = "ds-summary-text">
-        {{ summary(false) }}
-      </span>
-      <span
-        v-show="!$vuetify.breakpoint.smAndUp"
-        v-bind="{summary, calendar}"
-        name="shortDateSummary"
-        class = "ds-summary-text">
-        {{ summary(true) }}
-      </span>
-    
+    <v-flex
+      fluid
+      pb-3>
+      <v-layout
+        justify-space-between
+        row>
+        <v-flex>
```

Comment:
flex in flex:
Das äußere flex ist nach Zeilen angeordnet (Toolbar + Kalender)
Das innere flex ist nach Reihen angeordnet (Kalender-Buttons + Menü-Buttons) mit space-between, um die linken Buttons nach links und die rechten nach rechts auszurichten